src/components/TextArea/TextArea.test.tsx

```diff
@@ -5,7 +5,7 @@ import userEvent from '@testing-library/user-event'
 
 import { withAppContext } from 'test/utils'
 
-import TextArea from '..'
+import TextArea from './index'
```

Comment:
Linter should've complained about this. Should be `import TextArea from '.'`

Reply:
fixed